s3utils/localpath.go

```diff
@@ -43,6 +43,16 @@ func (p localPath) IsLocal() bool {
 	return true
 }
 
+// DirOrFolder returns "directory" for localPath and "folder" for s3Path
+func (p localPath) DirOrFolder() string {
+	return "directory"
+}
+
+// FileOrObject returns "object" for localPath and "file" for s3path
```

Comment:
this comment is reversed, right? file for localPath

Reply:
:man_facepalming: second time I've done this thanks!